array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,17 @@
+/* exported takeRight */
+/*
+output an array
+loop through array while counter is less than count arg AND array.length - i - 1 doesn't go below 0
+  start loop index from the end, then decrement at end of loop
+  unshift array@index
+return output
+ */
+
+function takeRight(array, count) {
+  var output = [];
+  for (var i = 0; i < count && (array.length - i - 1 >= 0); i++) {
```

Comment:
This is pretty complicated. I would just start the loop at `array.length - count`, that can help you simplify this by a lot.